adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -1224,9 +1224,14 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
 	// productionURL & sandBoxURL values are extracted from endpointConfig in api.yaml
 	endpointConfig := data.EndpointConfig
 
-	swagger.RateLimitLevel = data.RateLimitLevel
-	if swagger.RateLimitLevel == "API" {
-		swagger.RateLimitPolicy = data.RateLimitPolicy
+	if data.APIThrottlingPolicy != "" || data.ThrottlingLimit.Unit != "" {
+		logger.LoggerOasparser.Debugf("API level throttling limit found. Request count: %v, Unit: %v",
+			data.ThrottlingLimit.RequestCount, data.ThrottlingLimit.Unit)
+		swagger.RateLimitLevel = "API"
+		swagger.RateLimitPolicy = GetRLPolicyName(data.ThrottlingLimit.RequestCount, data.ThrottlingLimit.Unit)
+	} else if data.APIType == HTTP {
```

Comment:
Why do we check API type here to decide it is an Operation level rate limit policy?